restli-server/src/main/java/com/linkedin/restli/internal/server/model/RestLiAnnotationReader.java

```diff
@@ -2595,6 +2595,13 @@ private static void validateFinderMethod(final ResourceMethodDescriptor finderMe
     Method method = finderMethodDescriptor.getMethod();
     Class<?> valueClass = resourceModel.getValueClass();
 
+    if (valueClass == null)
+    {
+      throw new ResourceConfigException("@Finder method '" + method.getName()
+          + "' on class '" + resourceModel.getResourceClass().getName()
+          + "' has an invalid return type, we expect return type to be a record '");
```

Comment:
nit: A bit suggestion on the grammar:  

`we expect the return type to be 'Record' `

Otherwise looks good to me.

Reply:
I agree with Junchuan, however this seems to imply that the return type must be exactly `Record`. I'd suggest just copying the phrasing from other exceptions in this file: `The return type must be a RecordTemplate`.